src/ui/components/Video/imperative/MutableGraphicsState.tsx

```diff
@@ -86,6 +100,10 @@ export async function updateState(
       imageHeight: naturalHeight,
       imageWidth: naturalWidth,
     });
+    if (lock !== localLock) {
+      return;
```

Comment:
Nit: `fitImageToContainer()` isn't async so we can remove the `await` and this check.